src/ast2obj/builders/Installer.hx

```diff
@@ -20,6 +21,21 @@ class Installer {
         } else if (Sys.systemName() == "Mac") {
             ARDUINO_HOME = ARDUINO_HOME_OSX;
         }
+        var board_code = Sys.getEnv("TARGET_BOARD");
+        if (board_code == null) {
+            trace("Error, set env TARGET_BOARD.");
```

Comment:
So we _have_ to set a target board? Can we have it as a parameter too (rather than just an env var)? What about having some type of sensible "default board"?

Reply:
Hmm, i can move set target board to parameter.
> default board

Bad practice, probably need to look in direction of "project configuration file", for example as in platformio
![image](https://user-images.githubusercontent.com/13326808/101292461-a4355680-3820-11eb-93a7-6a02461cf135.png)

Reply:
Yeah, fair enough, maybe for now we can have it as env var _and_ param (with param taking precedence over env var?) 

We can look at project config later i think.